docs/worker-node/install-singularity.md

```diff
@@ -142,6 +142,24 @@ steps:
         ::console
         --security-opt seccomp=unconfined --security-opt systempaths=unconfined
 
+
+### Configuring Unprivileged Singularity ###
+
+When unprivileged singularity is enabled and VOs run singularity from
+CVMFS, the singularity configuration file also comes from CVMFS so local
+sites have no control over changing the configuration.  However, the
+most common local configuration change to the singularity RPM is to add
+additional local "bind path" options to map extra local file paths into
+containers.  This can instead be accomplished by setting the
+`SINGULARITY_BINDPATH` variable in the environment of jobs.  This is a
```

Comment:
```suggestion
`SINGULARITY_BINDPATH` variable in the environment of jobs through
[configuration](/other/configuration-with-osg-configure/#local-settings)
on your compute element.  This is a
```

Reply:
I'm going to change that to say that's one way to do it.